src/main/java/org/zaproxy/zap/extension/jwt/fuzzer/JWTFuzzAttackPopupMenuItem.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.jwt.fuzzer;
+
+import java.util.concurrent.TimeUnit;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.fuzz.ExtensionFuzz;
+import org.zaproxy.zap.extension.fuzz.FuzzerOptions;
+import org.zaproxy.zap.extension.fuzz.httpfuzzer.HttpFuzzer;
+import org.zaproxy.zap.extension.fuzz.httpfuzzer.HttpFuzzerHandler;
+import org.zaproxy.zap.extension.fuzz.messagelocations.MessageLocationsReplacementStrategy;
+import org.zaproxy.zap.view.messagecontainer.http.HttpMessageContainer;
+import org.zaproxy.zap.view.popup.PopupMenuItemHttpMessageContainer;
+
+/** @author KSASAN preetkaran20@gmail.com */
+public class JWTFuzzAttackPopupMenuItem extends PopupMenuItemHttpMessageContainer {
```

Comment:
Do we consider adding a new menu item or add jwt fuzzer as part of existing http fuzzer only ? Need to think on this. Actually if we add to existing http fuzzer then message replacer of JWT and http works differently hence requires changes in Fuzz.
![image](https://user-images.githubusercontent.com/13570884/83421705-25f99100-a446-11ea-936a-821ffcf7f8a3.png)
@kingthorin and @thc202 please help

Reply:
Adding a new menu item is fine. (You could precede it with a divider.)

Reply:
I don't think we should add another menu item, we have moved away from "menu item per configuration" by providing dialogues that allow the user to configure whatever it needs to be (assuming the current dialogue does).

Reply:
So @thc202 are you suggesting to use Fuzz menu item only ?

Reply:
Yes, the user can choose in the dialogue to fuzz the JWT token or plain HTTP message.

Reply:
Done. thanks alot.